interfaces/apparmor/template.go

```diff
@@ -468,6 +468,13 @@ var templateCommon = `
   /run/lock/snap.@{SNAP_INSTANCE_NAME}/ rw,
   /run/lock/snap.@{SNAP_INSTANCE_NAME}/** mrwklix,
 
+  # Allow all snaps to access each their own folder on the
+  # ubuntu-save partition, mark the ubuntu-save snap for read-only
+  # unless it's specifically its own snap directory.
+  /var/lib/snapd/save/snap/   r,
+  /var/lib/snapd/save/snap/** mrkix,
```

Comment:
I'm not sure we need this instead of fonlding the permissions with the one below, as we are not doing the bind mount done for the other directories for bind mounts, we are just using the instance name directly afaict. This bits need securiy review btw, maybe it should become its own PR

Reply:
Updated here and rebased on top of https://github.com/snapcore/snapd/pull/11933